test/run

```diff
@@ -197,6 +177,16 @@ test_application_with_user() {
   CONTAINER_ARGS="--user 12345" test_application
   check_result $?
 }
+
+function _cleanup() {
```

Comment:
Why cleanup is not used from `container-common-scripts`? Like in the other containers.

Reply:
That it because local `_cleanup` provides additional functionality that is not included in `ct_cleanup`. I plan to revise the `ct_cleanup` function in a near future and change this in all of the containers.